src/api/handlers/webhook.ts

```diff
@@ -404,8 +401,7 @@ export async function editSlashCommand(
   options: EditSlashCommandOptions,
   guildID?: string,
 ) {
-  // Use ... for content length due to unicode characters and js .length handling
-  if ([...options.name].length < 2 || [...options.name].length > 32) {
+  if (/^[\w-]{1,32}$/.test(options.name)) {
```

Comment:
We need to check if the name does not match the regex.
```suggestion
  if (!/^[\w-]{1,32}$/.test(options.name)) {
```